Ci Visibility: Improve documentation for tabs (horizontal) instead of vertical for pytest/pytest-benchmark/unittest (#21098)
DataDog/documentation: content/en/tests/setup/python.md

DD_SERVICE=my-python-app DD_ENV=ci pytest --ddtrace --ddtrace-patch-all
{{< /code-block >}}

### Adding custom tags to tests

> You can add custom tags to your tests by using the declaring `ddspan` as argument to your test:

Reviewer: ```suggestion
To add custom tags to your tests, declare `ddspan` as an argument in your test:
```

```python
from ddtrace import tracer

# Declare `ddspan` as argument to your test
def test_simple_case(ddspan):
    # Set your tags
    ddspan.set_tag("test_owner", "my_team")
    # test continues normally
    # ...
```

To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][5] section of the Python custom instrumentation documentation.

### Adding custom metrics to tests

> Just like tags, you can add custom metrics to your tests by using the current active span:

Reviewer: ```suggestion
Just like tags, to add custom metrics to your tests, use the current active span:
```

```python
from ddtrace import tracer

# Declare `ddspan` as an argument to your test
def test_simple_case(ddspan):
    # Set your tags
    ddspan.set_tag("memory_allocations", 16)
    # test continues normally
    # ...
```
Read more about custom metrics in the [Add Custom Metrics Guide][7].

> 

Reviewer: ```suggestion

[5]: /tracing/trace_collection/custom_instrumentation/python?tab=locally#adding-tags
[7]: /continuous_integration/guides/add_custom_metrics/?tab=python

```
Links need to be within the scope of the tab shortcode to render 👍
